src/gaxios.ts

```diff
@@ -148,7 +149,8 @@ export class Gaxios {
     opts.method = opts.method || 'GET';
 
     if (opts.params) {
-      const parts = new URL(opts.url);
+      const parts =
+          isBrowser() ? new window.URL(opts.url) : new URL.URL(opts.url);
```

Comment:
Instead of doing this every time through the call, at the entry point of the module can we just:

```
const URL = window ? window.URL :  URL;
```